fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/domain/Loan.java

```diff
@@ -5678,7 +5684,7 @@ private void updateLoanOutstandingBalaces() {
                         && !loanTransaction.isRepaymentAtDisbursement()) {
                     outstanding = outstanding.minus(loanTransaction.getAmount(getCurrency()));
                 } else {
-                    outstanding = outstanding.minus(loanTransaction.getPrincipalPortion(getCurrency()));
+                    outstanding = outstanding.plus(loanTransaction.getAmount(getCurrency()));
```

Comment:
@josehernandezfintecheandomx I think this change remained here unintentionally.

Reply:
Done!